lib/sycamore/sycamore/docset.py

```diff
@@ -115,13 +115,19 @@ def _truncate(s):
 
             pprint.pp(document, stream=stream)
 
-    def count(self, include_metadata=False, **kwargs) -> int:
+    def count(self, field: Optional[str] = None, include_metadata=False, **kwargs) -> int:
```

Comment:
Can you move the field arg after include_metadata?

that way if someone is currently doing `ds.count(True)` they won't break

Reply:
working on making a separate `count_distinct` instead